TOC.md

```diff
@@ -67,8 +67,8 @@
   + 扩容缩容
     - [使用 Ansible 扩容缩容](/how-to/scale/with-ansible.md)
   + 升级
-    - [使用 TiUP 升级（推荐）](/how-to/upgrade/using-tiup.md)
-    - [使用 TiDB Ansible 升级](/how-to/upgrade/from-previous-version.md)
+    - [使用 TiUP 升级（推荐）](/how-to/upgrade/tiup.md)
+    - [使用 TiDB Ansible 升级](/how-to/upgrade/ansible.md)
```

Comment:
I suggest not changing this file name if it's not a must, because soon we'll refactor the whole set of documentation, which means another round of file name changes. Let's try to reduce the times of changing file name. Just keep it a little bit more friendly to SEO.

Reply:
OK, I've changed back the file name.